src/chain-data-utils/bitquery.ts

```diff
@@ -30,7 +36,7 @@ async function _fetchAllPagesWithQueryAndVariables<T>(
   variables: any,
   dataPath: (data: any) => Array<T>
 ) {
-  const pageLimit = 2500;
+  const pageLimit = BITQUERY_FETCH_ALL_PAGES_PAGE_LIMIT();
```

Comment:
Any reason why these need to be functions? I think even as consts we should be able to mock them out for testing.